internal/pkg/deploy/cloudformation/pipeline_test.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const defaultImage = "aws/codebuild/amazonlinux2-x86_64-standard:3.0"
```

Comment:
nit: can we reuse `defaultPipelineBuildImage`?

Reply:
Sorry, I want to clarify. Do you mean rename `defaultImage` into `defaultPipelineBuildImage`? Or, using `defaultPipelineBuildImage` in internal/pkg/deploy/pipeline.go? This constant is in deploy pkg, but this test file is in cloudformation pkg. Do you intend to fix not internal/pkg/deploy/cloudformation/pipeline_test.go but l.152 in internal/pkg/deploy/pipeline_test.go?

Reply:
oh I see sorry that's my bad! you can ignore the comment then, I thought they were in the same pkg :)